INSTALL.rst

```diff
@@ -0,0 +1,352 @@
+  * `Latex <https://miktex.org/>`_ and `GhostScript <https://ghostscript.com/download/>`_ 
+    (for rendering text with LaTeX);
+
+.. note::
+
+   Matplotlib depends on a large number of non-python libraries. 
+   `pkg-config <https://www.freedesktop.org/wiki/Software/pkg-config/>`__
+   can be used to find required non-python libraries and thus make install go
+   more smoothly if the libraries and headers are not in the expected
+   locations.
+
+.. note::
+
+  The following libraries are shipped with Matplotlib:
+
+    - `Agg`: the antigrain C++ rendering engine;
+    - `qhull`: to compute Delaunay triangulation;
+    - `ttconv`: a true type font utility.
+
+.. _build_linux:
+
+Building on Linux
+-----------------
+
+It is easiest to use your system package manager to install the dependencies.
+
+If you are on Debian/Ubuntu, you can get all the dependencies
+required to build Matplotlib with::
+
+   sudo apt-get build-dep python-matplotlib
+
+If you are on Fedora/RedHat, you can get all the dependencies required
+to build Matplotlib by first installing ``yum-builddep`` and then
+running::
+
+   su -c "yum-builddep python-matplotlib"
+
+This does not build Matplotlib, but it does get and install the
+build dependencies, which will make building from source easier.
+
+
+.. _build_osx:
+
+Building on OSX
+---------------
+
+The build situation on OSX is complicated by the various places one
+can get the libpng and freetype requirements (darwinports, fink,
+/usr/X11R6) and the different architectures (e.g., x86, ppc, universal),
+the different OSX version (e.g., 10.4 and 10.5). We recommend that you build
+the way we do for the OSX release: get the source from the tarball or the
+git repository and install the required dependencies through a third-party
+package manager. Two widely used package managers are homebrew, and MacPorts.
+The following example illustrates how to install libpng and freetype using
+brew::
+
+  brew install libpng freetype pkg-config
+
+If you are using MacPorts, execute the following instead::
+
+  port install libpng freetype pkgconfig
+
+To install Matplotlib from source, execute::
+
+  python setup.py install
+
+
+Note that your environment is somewhat important. Some conda users have 
+found that, to run the tests, their PYTHONPATH must include 
+/path/to/anaconda/.../site-packages and their DYLD_FALLBACK_LIBRARY_PATH 
+must include /path/to/anaconda/lib.
+
+
+.. _build_windows:
+
+Building on Windows
+-------------------
+
+The Python shipped from https://www.python.org is compiled with Visual Studio
+2008 for versions before 3.3, Visual Studio 2010 for 3.3 and 3.4, and
+Visual Studio 2015 for 3.5 and 3.6.  Python extensions are recommended to be compiled
+with the same compiler.
+
+Since there is no canonical Windows package manager, the methods for building
+freetype, zlib, and libpng from source code are documented as a build script
+at `matplotlib-winbuild <https://github.com/jbmohler/matplotlib-winbuild>`_.
```

Comment:
I am actually slightly worried that this repository is totally out of date. It hasn't bee updated in 2014, so there is maybe a bit of work to be done to have the scripts run.